z2/src/chain/node.rs

```diff
@@ -364,20 +364,21 @@ impl ChainNode {
         self.import_config_files().await?;
         self.run_provisioning_script().await?;
 
+        // TODO implement a more effective check
```

Comment:
What doesn't work with this check?

Reply:
network as prototestnet or protomainnet after upgrade, nodes do not provide responses for a time between 6 to 10 minutes making this check de facto failing and not useful